infra/lambda/lambda_function.py

```
@@ -18,6 +18,187 @@
+def _get_bedrock_api_key() -> Optional[str]:
+    global _bedrock_key_cache
+    if _bedrock_key_cache:
+        return _bedrock_key_cache
+    try:
+        resp = _ssm_client.get_parameter(
+            Name="/iam-dashboard/dev/bedrock-api-key",
+            WithDecryption=True,
+        )
+        _bedrock_key_cache = resp["Parameter"]["Value"]
+        return _bedrock_key_cache
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)